pkg/osbuild/gzip_stage.go

```diff
@@ -0,0 +1,41 @@
+package osbuild
+
+type GzipStageOptions struct {
+	// Filename for gz archive
+	Filename string `json:"filename"`
+	Level    *int   `json:"level,omitempty"`
```

Comment:
AFAICT this is not used or tested, maybe we can remove it and add it when we need it (it seems xz, zstd also do not expose the compression level?). Unless of course we will use it soon.

Reply:
If we don't use it and don't have plans to use it I would be (strongly) in favor of YAGNI and just not do it (for now).